HaaLimits/python/HaaLimits.py

```diff
@@ -71,138 +73,122 @@ def initializeWorkspace(self):
         self.addX(*self.XRANGE,unit='GeV',label=self.XLABEL)
         self.addMH(*self.SPLINERANGE,unit='GeV',label=self.SPLINELABEL)
 
-    def buildModel(self,region='PP',**kwargs):
+    def buildModel(self, region='PP', addUpsilon=False, setUpsilonLambda=False, sidesModeled=False, **kwargs):
```

Comment:
addUpsilon default to True
sidesModeled default to True

Reply:
Nevermind, sideModelled default to False,
but it needs a better name, it is not clear what it is meant to do,
If false, it disables Jpsi and Upsilon peaks